src/test/task.test.js

```diff
@@ -0,0 +1,31 @@
+import { addTask, removeTask } from '../module/task';
+
+jest.mock('../module/storage.js');
+
+let tasks = [
+  { description: 'do some coding', completed: false, index: 1 },
+  { description: 'hire a help', completed: true, index: 2 },
+];
+
+describe('Add Tasks', () => {
```

Comment:
Try to add a more descriptive description for your project. It helps testers to understand about your test is for.